### ISSUE TITLE
1.1.0.post20240409 provides a buggy CMake package

### ISSUE THREAD

**Issue by kou** [conda-forge/rapidjson-feedstock#8]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

RapidJSON has a CMake related problem: https://github.com/Tencent/rapidjson/issues/2192

It causes a build error of apache/arrow: https://github.com/apache/arrow/issues/41509

Could you apply https://github.com/Tencent/rapidjson/pull/2193 as a work around?

### Installed packages

```shell
https://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/builds/49736306#L532

C:\projects\arrow>conda list -n arrow 
# packages in environment at C:\Miniconda38-x64\envs\arrow:
#
# Name                    Version                   Build  Channel
aiobotocore               2.12.2             pyhd8ed1ab_0    conda-forge
aiohttp                   3.9.5           py310h8d17308_0    conda-forge
aioitertools              0.11.0             pyhd8ed1ab_0    conda-forge
aiosignal                 1.3.1              pyhd8ed1ab_0    conda-forge
async-timeout             4.0.3              pyhd8ed1ab_0    conda-forge
attrs                     23.2.0             pyh71513ae_0    conda-forge
aws-c-auth                0.7.3                hd125877_3    conda-forge
aws-c-cal                 0.6.2                h04c9df6_1    conda-forge
aws-c-common              0.9.0                hcfcfb64_0    conda-forge
aws-c-compression         0.2.17               h04c9df6_2    conda-forge
aws-c-event-stream        0.3.2                h495bb32_0    conda-forge
aws-c-http                0.7.12               h0890e15_1    conda-forge
aws-c-io                  0.13.32              h83b3346_3    conda-forge
aws-c-mqtt                0.9.5                h0fd1aac_1    conda-forge
aws-c-s3                  0.3.14               h9f49523_3    conda-forge
aws-c-sdkutils            0.1.12               h04c9df6_1    conda-forge
aws-checksums             0.1.17               h04c9df6_1    conda-forge
aws-crt-cpp               0.23.0               hf4e30cf_5    conda-forge
aws-sdk-cpp               1.11.68             hf794bbe_13    conda-forge
azure-core-cpp            1.11.1               h249a519_1    conda-forge
azure-identity-cpp        1.6.0                h91493d7_1    conda-forge
azure-storage-blobs-cpp   12.10.0              h91493d7_1    conda-forge
azure-storage-common-cpp  12.5.0               h91493d7_4    conda-forge
azure-storage-files-datalake-cpp 12.9.0               h91493d7_1    conda-forge
backports.zoneinfo        0.2.1           py310h5588dad_8    conda-forge
benchmark                 1.8.3                h63175ca_0    conda-forge
boost-cpp                 1.84.0               h6f18f0d_2    conda-forge
boto3                     1.34.51            pyhd8ed1ab_0    conda-forge
botocore                  1.34.51         pyge310_1234567_0    conda-forge
brotli                    1.1.0                hcfcfb64_1    conda-forge
brotli-bin                1.1.0                hcfcfb64_1    conda-forge
brotli-python             1.1.0           py310h00ffb61_1    conda-forge
bzip2                     1.0.8                hcfcfb64_5    conda-forge
c-ares                    1.28.1               hcfcfb64_0    conda-forge
ca-certificates           2024.2.2             h56e8100_0    conda-forge
ccache                    4.9.1                h83c9fb6_0    conda-forge
cffi                      1.16.0          py310h8d17308_0    conda-forge
clang                     14.0.6               h8e541a6_1    conda-forge
clang-14                  14.0.6          default_h66ee7f4_1    conda-forge
clang-format              14.0.6          default_h66ee7f4_1    conda-forge
clang-tools               14.0.6          default_h66ee7f4_1    conda-forge
clangdev                  14.0.6          default_h66ee7f4_1    conda-forge
clangxx                   14.0.6          default_h66ee7f4_1    conda-forge
click                     8.1.7           win_pyh7428d3b_0    conda-forge
cloudpickle               3.0.0              pyhd8ed1ab_0    conda-forge
cmake                     3.29.2               hf0feee3_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
cython                    3.0.10          py310h00ffb61_0    conda-forge
exceptiongroup            1.2.0              pyhd8ed1ab_2    conda-forge
frozenlist                1.4.1           py310h8d17308_0    conda-forge
fsspec                    2024.3.1           pyhca7485f_0    conda-forge
gflags                    2.2.2             ha925a31_1004    conda-forge
glog                      0.7.0                h9cd36e5_0    conda-forge
gmock                     1.14.0               h57928b3_1    conda-forge
google-cloud-cpp          2.5.0                hbd02b2f_0    conda-forge
grpc-cpp                  1.49.1               h9c18f36_1    conda-forge
gtest                     1.14.0               h91493d7_1    conda-forge
hypothesis                6.100.2            pyha770c72_0    conda-forge
idna                      3.7                pyhd8ed1ab_0    conda-forge
iniconfig                 2.0.0              pyhd8ed1ab_0    conda-forge
jmespath                  1.0.1              pyhd8ed1ab_0    conda-forge
krb5                      1.21.2               heb0366b_0    conda-forge
libabseil                 20220623.0      cxx17_h1a56200_6    conda-forge
libblas                   3.9.0           22_win64_openblas    conda-forge
libboost                  1.84.0               hcc118f5_2    conda-forge
libboost-devel            1.84.0               h91493d7_2    conda-forge
libboost-headers          1.84.0               h57928b3_2    conda-forge
libbrotlicommon           1.1.0                hcfcfb64_1    conda-forge
libbrotlidec              1.1.0                hcfcfb64_1    conda-forge
libbrotlienc              1.1.0                hcfcfb64_1    conda-forge
libcblas                  3.9.0           22_win64_openblas    conda-forge
libclang                  14.0.6          default_h77d9078_1    conda-forge
libclang-cpp              14.0.6          default_hf9e4a47_1    conda-forge
libclang13                14.0.6          default_h77d9078_1    conda-forge
libcrc32c                 1.1.2                h0e60522_0    conda-forge
libcurl                   8.7.1                hd5e4a3a_0    conda-forge
libevent                  2.1.12               h3671451_1    conda-forge
libexpat                  2.6.2                h63175ca_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libflang                  5.0.0           h6538335_20180525    conda-forge
libglib                   2.80.0               h39d0aa6_6    conda-forge
libgoogle-cloud           2.5.0                hbd02b2f_0    conda-forge
libgrpc                   1.49.1               h6a6baca_1    conda-forge
libhiredis                1.0.2                h0e60522_0    conda-forge
libiconv                  1.17                 hcfcfb64_2    conda-forge
libintl                   0.22.5               h5728263_2    conda-forge
liblapack                 3.9.0           22_win64_openblas    conda-forge
libllvm14                 14.0.6               h97333cc_4    conda-forge
libopenblas               0.3.27          pthreads_hc140b1d_0    conda-forge
libprotobuf               3.21.12              h12be248_2    conda-forge
libsqlite                 3.45.3               hcfcfb64_0    conda-forge
libssh2                   1.11.0               h7dfc565_0    conda-forge
libthrift                 0.20.0               ha2b3283_0    conda-forge
libutf8proc               2.8.0                h82a8f57_0    conda-forge
libuv                     1.48.0               hcfcfb64_0    conda-forge
libxml2                   2.12.6               hc3477c8_2    conda-forge
libzlib                   1.2.13               hcfcfb64_5    conda-forge
llvm-meta                 5.0.0                         0    conda-forge
llvm-tools                14.0.6               hac7d729_4    conda-forge
llvmdev                   14.0.6               h97333cc_4    conda-forge
lz4-c                     1.9.4                hcfcfb64_0    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
make                      4.3                  h3d2af85_1    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
multidict                 6.0.5           py310h8d17308_0    conda-forge
ninja                     1.12.0               h91493d7_0    conda-forge
nlohmann_json             3.11.3               h1537add_0    conda-forge
nodejs                    20.12.2              h57928b3_0    conda-forge
nomkl                     1.0                  h5ca1d4c_0    conda-forge
numpy                     1.26.4          py310hf667824_0    conda-forge
openmp                    5.0.0                    vc14_1    conda-forge
openssl                   3.3.0                hcfcfb64_0    conda-forge
orc                       1.9.0                hada7b9e_1    conda-forge
packaging                 24.0               pyhd8ed1ab_0    conda-forge
pandas                    2.2.2           py310hecd3228_0    conda-forge
pcre2                     10.43                h17e33f8_0    conda-forge
pip                       24.0               pyhd8ed1ab_0    conda-forge
pkg-config                0.29.2            h2bf4dc2_1008    conda-forge
pluggy                    1.5.0              pyhd8ed1ab_0    conda-forge
pycparser                 2.22               pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1              pyh0701188_6    conda-forge
pytest                    8.2.0              pyhd8ed1ab_0    conda-forge
pytest-faulthandler       2.0.1                      py_0    conda-forge
python                    3.10.14         h4de0772_0_cpython    conda-forge
python-dateutil           2.9.0              pyhd8ed1ab_0    conda-forge
python-tzdata             2024.1             pyhd8ed1ab_0    conda-forge
python_abi                3.10                    4_cp310    conda-forge
pytz                      2024.1             pyhd8ed1ab_0    conda-forge
rapidjson                 1.1.0.post20240409      h63175ca_0    conda-forge
re2                       2022.06.01           h0e60522_1    conda-forge
s3fs                      2024.3.1           pyhd8ed1ab_0    conda-forge
s3transfer                0.10.1             pyhd8ed1ab_0    conda-forge
setuptools                69.5.1             pyhd8ed1ab_0    conda-forge
setuptools-scm            8.0.4              pyhd8ed1ab_1    conda-forge
setuptools_scm            8.0.4                hd8ed1ab_1    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snappy                    1.1.10               hfb803bf_1    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
thrift-compiler           0.20.0               h63175ca_0    conda-forge
thrift-cpp                0.20.0               h57928b3_0    conda-forge
tk                        8.6.13               h5226925_1    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
typing-extensions         4.11.0               hd8ed1ab_0    conda-forge
typing_extensions         4.11.0             pyha770c72_0    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
urllib3                   2.0.7              pyhd8ed1ab_0    conda-forge
vc                        14.3                hcf57466_18    conda-forge
vc14_runtime              14.38.33130         h82b7239_18    conda-forge
vs2015_runtime            14.38.33130         hcb4865c_18    conda-forge
wheel                     0.43.0             pyhd8ed1ab_1    conda-forge
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
wrapt                     1.16.0          py310h8d17308_0    conda-forge
xsimd                     12.1.1               h91493d7_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
yarl                      1.9.4           py310h8d17308_0    conda-forge
zlib                      1.2.13               hcfcfb64_5    conda-forge
zstd                      1.5.5                h12be248_0    conda-forge
```

### Environment info

```shell
https://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/builds/49736306#L36

C:\projects\arrow>conda info -a 
     active environment : None
       user config file : C:\Users\appveyor\.condarc
 populated config files : C:\Users\appveyor\.condarc
          conda version : 4.8.2
    conda-build version : not installed
         python version : 3.8.1.final.0
       virtual packages : 
       base environment : C:\Miniconda38-x64  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Miniconda38-x64\pkgs
                          C:\Users\appveyor\.conda\pkgs
                          C:\Users\appveyor\AppData\Local\conda\conda\pkgs
       envs directories : C:\Miniconda38-x64\envs
                          C:\Users\appveyor\.conda\envs
                          C:\Users\appveyor\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.8.1 Windows/10 Windows/10.0.17763
          administrator : True
             netrc file : None
           offline mode : False
# conda environments:
#
base                  *  C:\Miniconda38-x64
sys.version: 3.8.1 (default, Mar  2 2020, 13:06:26) [...
sys.prefix: C:\Miniconda38-x64
sys.executable: C:\Miniconda38-x64\python.exe
conda location: C:\Miniconda38-x64\lib\site-packages\conda
conda-build: None
conda-env: C:\Miniconda38-x64\Scripts\conda-env.exe
user site dirs: 
ANDROID_NDK_PATH: C:\Microsoft\AndroidNDK64\android-ndk-r16b
APR_ICONV_PATH: C:\Program Files (x86)\Subversion\iconv
CIO_TEST: <not set>
CONDA_ROOT: C:\Miniconda38-x64
GIT_LFS_PATH: C:\Program Files\Git LFS
HOMEPATH: \Users\appveyor
PATH: C:\Miniconda38-x64;C:\Miniconda38-x64\Scripts;C:\Miniconda38-x64\Library\bin;C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\Program Files\Git\cmd;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\PowerShell\7\;C:\Program Files\7-Zip;C:\Program Files\Microsoft\Web Platform Installer\;C:\Tools\NuGet;C:\Tools\PsTools;C:\Program Files\Git\usr\bin;C:\Program Files\Git LFS;C:\Program Files\Mercurial\;C:\Program Files (x86)\Subversion\bin;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\dotnet\;C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\Extensions\Microsoft\SQLDB\DAC\150;C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin;C:\Tools\xUnit;C:\Tools\xUnit20;C:\Tools\NUnit\bin;C:\Tools\NUnit3\bin;C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\Extensions\TestPlatform;C:\Ruby32\bin;C:\Tools\WebDriver;C:\Python312;C:\Python312\Scripts;C:\Program Files (x86)\nodejs\;C:\Program Files\nodejs;C:\Program Files (x86)\iojs;C:\Program Files\iojs;C:\Users\appveyor\AppData\Roaming\npm;C:\Program Files (x86)\Yarn\bin\;C:\go\bin;C:\Program Files\Java\jdk1.8.0\bin;C:\Program Files\erl10.7\bin;C:\flutter\bin;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\140\Tools\Binn\;C:\Program Files\Microsoft SQL Server\140\Tools\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Amazon\AWSCLI\bin\;C:\Program Files\Amazon\AWSCLI;C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\bin;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\Microsoft DirectX SDK;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files (x86)\Microsoft SQL Server\110\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\120\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\150\DTS\Binn\;C:\Tools\Doxygen;C:\Tools\Graphviz\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\bin;C:\Program Files\LLVM\bin;C:\Tools\vcpkg;C:\Tools\Coverity\bin;C:\Program Files (x86)\NSIS;C:\Tools\Octopus;C:\Program Files\Meson\;C:\Program Files (x86)\Apache\Maven\bin;C:\Tools\GitVersion;C:\Users\appveyor\AppData\Local\Microsoft\WindowsApps;C:\Users\appveyor\.dotnet\tools;C:\Users\appveyor\AppData\Roaming\npm;C:\Users\appveyor\AppData\Local\Yarn\bin;C:\Program Files\AppVeyor\BuildAgent\
PSMODULEPATH: C:\Users\appveyor\Documents\WindowsPowerShell\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules;C:\Program Files\AppVeyor\BuildAgent\Modules;C:\Users\appveyor\Documents\WindowsPowerShell\Modules;C:\Program Files (x86)\Microsoft SQL Server\140\Tools\PowerShell\Modules\;C:\Program Files (x86)\Microsoft SQL Server\150\Tools\PowerShell\Modules\;C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\platform\PowerShell
PYTHON: 3.10
REQUESTS_CA_BUNDLE: <not set>
SSL_CERT_FILE: <not set>
```

**Comment by zklaus** [comment]
I think this was addressed in #9 and can be closed.

**Comment by kou** [comment]
Thanks.
I've confirmed that #9 addressed this. I close this.
